olive/auto_optimizer/config_template/precision_aware_pass_flows.yaml

```diff
@@ -0,0 +1,16 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+
+
+# Passes which can conduct precision aware optimization
+# e.g. OrtTransformersOptimization can be applied to fp32, fp16
+# but OnnxQuantization only works for int8
+OrtTransformersOptimization:
+  - fp32
+  - fp16
+
```

Comment:
Why do we need a separate .yaml here? Just like OnnxMatMul4Quantizer lists 2 EPs in pass_capability.yaml the passes can list multiple precisions.

Reply:
That is useless config from before's implementation, will remove it.